src/server/services/wms/qgswmsparameters.h

```diff
@@ -192,6 +192,15 @@ namespace QgsWms
        */
       bool isValid() const override;
 
+      /**
+       * Converts the parameter into a list of strings and keeps empty parts
+       * Default style value is an empty string
+       * \param delimiter The character used for delimiting
+       * \returns A list of strings
+       * \since QGIS 3.8
```

Comment:
@pblottiere @elpaso @nyalldawson
Do I have to update \since before merge ?
Do I add a commit to the branch or do I update the commit (commit, rebase, push force) ?

Reply:
You should have done the reverse, to add `3.4.7` in the master branch. For instance https://github.com/qgis/QGIS/pull/9480 was for master

Reply:
ok, So I do a new PR in master just to fix it ?

Reply:
Well, it has already been merged in master, so I guess you need this new PR now. Next time if you know you will backport, do it before ;-)

Reply:
thanks @Gustry